pyproject.toml

```diff
@@ -60,10 +60,10 @@ docs = [
 ]
 
 [project.entry-points."aiida.calculations"]
-"lammps.base" = "aiida_lammps.calculations.lammps.base:BaseLammpsCalculation"
+"lammps.base" = "aiida_lammps.calculations.lammps.base:LammpsBaseCalculation"
 "lammps.combinate" = "aiida_lammps.calculations.lammps.combinate:CombinateCalculation"
-"lammps.force" = "aiida_lammps.calculations.lammps.force:ForceCalculation"
-"lammps.md" = "aiida_lammps.calculations.lammps.md:MdCalculation"
+"lammps.force" = "aiida_lammps.calculations.lammps.force:LammpsForceCalculation"
+"lammps.md" = "aiida_lammps.calculations.lammps.md:LammpsMdCalculation"
 "lammps.md.multi" = "aiida_lammps.calculations.lammps.md_multi:MdMultiCalculation"
 "lammps.optimize" = "aiida_lammps.calculations.lammps.optimize:OptimizeCalculation"
```

Comment:
What about these and the `CombinateCalculation`?

Reply:
Thanks I have missed those ones!
Should I also do the same in the parsers? or make a different PR for that?

Reply:
Yeah I would also add the parsers to this PR.

Reply:
On a side note: do you have a specific reason for merging pull requests, instead of squash merging? Or are you just using it because it is the default. If you are just using it by default, may I suggest we start using squash merge instead, at least for PRs that don't have neatly crafted commits that should be kept separate? This keeps the history on the main branch more linear and a lot more easy to navigate. These merge commits and intermediate commits from feature branches are almost never useful. We are using this approach in all other `aiidateam` repos as well and works really well.

Reply:
Basically because it is the default.
With squash merging you refer to squashing all the merges in a PR into a single commit right? which in principle could have been many commits in the fork. Or is it something else?
I think that keeping the main branch history clean would be very nice I agree. 
Any thoughts on this @chrisjsewell ?

Reply:
> With squash merging you refer to squashing all the merges in a PR into a single commit right?

Indeed. Next to the `Merge pull request` button is a little arrow where you can select the merge method. There you select "Squash and merge" instead and then click the button. It will open a text editor to format the final commit message. By default it takes a concatanation of commit messages of all commits. I usually edit this to be a single nice final commit message.

Chris is on a holiday, so not sure if he will see this. But I am pretty sure he also uses squash merge (where applicable). Note that there are still cases in which you don't want to squash, but in that case, the PR author needs to properly prepare the commits in the PR branch as they should be merged into the main branch. Then we use rebase and merge instead of squash. But for most PRs, jsut a single commit is sufficient and so we simply squash.

Reply:
Nice! I'll do it then for this PR, I normally did the squashing by hand, but if github can do it for me I'm happy about it.